runelite-api/src/main/java/net/runelite/api/Client.java

```diff
@@ -59,6 +59,10 @@
 
 	void setUsername(String name);
 
+	void setPassword(String password);
+
+	int getCurrentLoginField();
```

Comment:
These methods would benefit from some Javadocs imo, because of how sensitive they look.

Reply:
looks like they've got JavaDoc now